microsoft-edge/webview2/concepts/print.md

```diff
@@ -0,0 +1,736 @@
+---
+title: Printing in WebView2 apps
+description: How to print a web page in a WebView2 app.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 01/12/2023
+---
+# Print
+
+There are a few different ways to print a web page in WebView2 which give you varying levels of ease of use and control.
+
+## ShowPrintUI
+
+The ShowPrintUI show the WebView2 print preview dialog or OS print dialog for the current top-level document in the WebView2. With this mechanism you easily get a printing experience the user will be familiar with.
+
+<!-- ------------------------------ -->
+
+# [C#](#tab/csharp)
+
+[CoreWebView2.ShowPrintUI() Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.showprintui#microsoft-web-webview2-core-corewebview2-showprintui)
+[CoreWebView2.ShowPrintUI(CoreWebView2PrintDialogKind) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.showprintui#microsoft-web-webview2-core-corewebview2-showprintui(microsoft-web-webview2-core-corewebview2printdialogkind))
+[CoreWebView2PrintDialogKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2printdialogkind)
+
+
+<!-- ------------------------------ -->
+
+# [C++](#tab/cpp)
+
+[ICoreWebView2_16::ShowPrintUI(printDialogKind) Method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#showprintui)
+[COREWEBVIEW2_PRINT_DIALOG_KIND Enum](/microsoft-edge/webview2/reference/win32/icorewebview2_16#corewebview2_print_dialog_kind)
+
+
+---
+
+<!-- end of tab-set -->
+
+### Example: ShowPrintUI
```

Comment:
Global in .md: heading levels:

On the one hand, try hard to make everything an h2, b/c "In this article" nav in upper right only lists h2s.

Other the other hand, Example sections sometimes are ok to hide from Nav by demoting to h3 which we prefer to tag as h4 to get a big difference in font size.

Recommend using only h2/h4/h6, esp. h2.

Reply:
Changed to use h4 instead of h3.